include/ur_api.h

```diff
@@ -3212,12 +3217,14 @@ urDevicePartition(
 ///     - ::UR_RESULT_ERROR_INVALID_NULL_POINTER
 ///         + `NULL == ppBinaries`
 ///         + `NULL == pSelectedBinary`
+///     - ::UR_RESULT_ERROR_INVALID_SIZE
 UR_APIEXPORT ur_result_t UR_APICALL
 urDeviceSelectBinary(
     ur_device_handle_t hDevice,                     ///< [in] handle of the device to select binary for.
     const uint8_t** ppBinaries,                     ///< [in] the array of binaries to select from.
-    uint32_t NumBinaries,                           ///< [in] the number of binaries passed in ppBinaries. Must greater than or
-                                                    ///< equal to zero.
+    uint32_t NumBinaries,                           ///< [in] the number of binaries passed in ppBinaries. 
+                                                    ///< Must greater than or equal to zero otherwise
+                                                    ///< ::UR_RESULT_ERROR_INVALID_SIZE is returned.
```

Comment:
shouldn't `UR_RESULT_ERROR_INVALID_SIZE` be the interpretation of 0 value? as from my understanding to this from [ur current specification](https://oneapi-src.github.io/unified-runtime/core/api.html#_CPPv416ur_device_info_t). it says:
> enumerator UR_RESULT_ERROR_INVALID_SIZE
 [Validation] size argument is invalid (e.g., must not be zero)

Reply:
That's a good point. UR_RESULT_ERROR_INVALID_VALUE probably makes more sense?

Reply:
Done.